### PR TITLE
Routine Bundle Update

### PR DESCRIPTION
**Pull request by jedschneider**
### What was the problem or feature?

I got this when trying to bundle update the broker_suite in lambas, I figure this is better to solve upstream and remove the dependencies directly from broker_suite? Or, at least any pinned versions:

```
============================================================================
= WARNING: You do not want to run this version of Standard!                =
============================================================================

What's going on here?
---------------------
Version 1.35.0 of Standard was set to depend on `~> 1.62' of RuboCop. This
constraint is too loose, and covers all minor versions of RuboCop 1.x.

"How do I fix this?", you might be asking.

How to fix this
---------------
If you add a version specifier pinning `standard' to a version newer
than 1.35.1, Bundler will resolve appropriate versions of `standard',
`rubocop', and any other rubocop-dependent gems you may have installed.

1. Update your Gemfile to pin standard to be at least one such version:

  gem "standard", ">= 1.35.1"

2. Run `bundle`, which will install and lock more appropriate versions

  Example output:
    Using rubocop 1.48.1 (was 1.49.0)
    Using standard 1.26.0 (was 0.0.36)

This version (1.35.0.1) is an inoperative placeholder gem that exists
solely to print this message.

We're very sorry for this inconvenience!

============================================================================
=                         END OF BIG SCARY WARNING                         =
============================================================================
```

Also, I'm not sure the use of standard and rubocop? I don't recall the history there.

### What was the solution?

- [x] Security Impact has been considered (if yes please describe)
- [x] Network Impacts have been considered (if yes please describe)

### CODE REVIEW

File: livery.gemspec [hunk 1]
```diff
@@ -23,11 +23,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "betterlint", "~> 1.4"
-  spec.add_runtime_dependency "rubocop", "~> 1"
+  spec.add_runtime_dependency "rubocop", "~> 1.63.5"
   spec.add_runtime_dependency "rubocop-gitlab-security", "~> 0.1"
   spec.add_runtime_dependency "rubocop-rails", "~> 2"
   spec.add_runtime_dependency "rubocop-rspec", "~> 2"
-  spec.add_runtime_dependency "standard", "~> 1"
+  spec.add_runtime_dependency "standard", "~> 1.36.0"
```
Comment (plukevdh) [inline livery.gemspec:29]: why do we have both?

**Review by plukevdh — APPROVED** [review]

File: livery.gemspec (hunk 1 above)
Comment (jedschneider) [reply livery.gemspec:29]: I'm honestly not sure...? I know standard is like a superset of rubocop, and so its like a "yes standard, and additional rubocops".

Comment (plukevdh) [reply livery.gemspec:29]: I guess i thought it was an alternative...

Comment (jedschneider) [reply livery.gemspec:29]: I think more like this: https://www.fastruby.io/blog/ruby/code-quality/how-we-use-rubocop-and-standardrb.html

speaking of which though, maybe we should move anything rails based to the only projects using rails.

Comment (jedschneider) [reply livery.gemspec:29]: @jayzes this occurs to me now this is a public repository, do you recall the interplay between standardrb and rubocop?

**Pull request merged**